doc/modules/reference.rst

```diff
@@ -229,20 +229,20 @@ the :ref:`user guide <user_guide>` for more information and usage examples.
    load_confounds
    load_confounds_strategy
 
-.. _io_ref:
+.. _maskers_ref:
```

Comment:
Did you replace all uses of `io_ref` throughout the documentation with `maskers_ref` too?

Reply:
Thanks for checking! I couldn't find any. Also, the documentation full build runs in strict mode and would have crashed if there were broken references somewhere.